bin/insight

```diff
@@ -10,15 +10,21 @@
  * file that was distributed with this source code.
  */
 
-// installed via composer?
-if (file_exists($a = __DIR__.'/../../../autoload.php')) {
-    require_once $a;
-} else {
-    require_once __DIR__.'/../vendor/autoload.php';
-}
-
 use SensioLabs\Insight\Cli\Application;
 
-$application = new Application();
+if (false === in_array(PHP_SAPI, ['cli', 'phpdbg', 'embed'], true)) {
```

Comment:
`embed` can be omitted if you wish, but at least `cli` and `phpdbg` should be put